content/en/blog/2017/0.1-using-network-policy/index.md

```diff
@@ -29,7 +29,7 @@ In contrast, operating at the network layer has the advantage of being universal
 
 ## Implementation
 
-The Istio’s proxy is based on [Envoy](https://envoyproxy.github.io/envoy/), which is implemented as a user space daemon in the data plane that
+The Istio’s proxy is based on [Envoy](https://github.com/envoyproxy/envoy), which is implemented as a user space daemon in the data plane that
```

Comment:
Since this doc is so old, this fix isn't really needed. The more important question is why isn't the broken link detected by the lint checker?

@arijitvt maybe you'd like to track that down?

Reply:
Did this pr as part of the Istio day. Should I file an issue that the linter did not check this?

Reply:
Hi @frankbu we were doing the contribfest session at Kubecon giving smaller tasks to audience to fix some easy issues. Our linter currently has EXTERNAL_LINK check disabled and that is why it is not caught

Reply:
Thanks @kfaseela